--- minecraft/src/net/minecraft/client/Minecraft.java	2018-01-21 00:00:46.000000000 -0800
+++ src/main/java/net/minecraft/client/Minecraft.java	2018-01-21 00:13:01.567974119 -0800
@@ -192,7 +192,7 @@
       this.serverPort = var2;
    }
 
-   public void startGame() {
+   public void startGame() throws LWJGLException {
       if (this.mcCanvas != null) {
          Graphics var1 = this.mcCanvas.getGraphics();
          if (var1 != null) {
@@ -302,7 +302,7 @@
 
    }
 
-   private void loadScreen() {
+   private void loadScreen() throws LWJGLException {
       ScaledResolution var1 = new ScaledResolution(this.gameSettings, this.displayWidth, this.displayHeight);
       GL11.glClear(16640);
       GL11.glMatrixMode(5889);
@@ -358,7 +358,7 @@
    }
 
    public static File getAppDir(String var0) {
-      // $FF: Couldn't be decompiled
+      return new File("./." + var0);
    }
 
    private static EnumOS2 getOs() {
