--- minecraft/src/net/minecraft/unknown/ChunkFilePattern.java	2018-01-21 00:00:36.000000000 -0800
+++ src/main/java/net/minecraft/unknown/ChunkFilePattern.java	2018-01-21 12:57:29.825691663 -0800
@@ -8,7 +8,7 @@
 class ChunkFilePattern implements FilenameFilter {
    public static final Pattern field_22189_a = Pattern.compile("c\\.(-?[0-9a-z]+)\\.(-?[0-9a-z]+)\\.dat");
 
-   private ChunkFilePattern() {
+   public ChunkFilePattern() {
    }
 
    public boolean accept(File var1, String var2) {
