--- minecraft/src/net/minecraft/unknown/Chunk.java	2018-01-21 00:00:30.000000000 -0800
+++ src/main/java/net/minecraft/unknown/Chunk.java	2018-01-21 00:32:52.976625422 -0800
@@ -18,7 +18,7 @@
    public int lowestBlockHeight;
    public final int xPosition;
    public final int zPosition;
-   public Map chunkTileEntityMap;
+   public Map<ChunkPosition, TileEntity> chunkTileEntityMap;
    public List[] entities;
    public boolean isTerrainPopulated;
    public boolean isModified;
@@ -27,7 +27,7 @@
    public long lastSaveTime;
 
    public Chunk(World var1, int var2, int var3) {
-      this.chunkTileEntityMap = new HashMap();
+      this.chunkTileEntityMap = new HashMap<>();
       this.entities = new List[8];
       this.isTerrainPopulated = false;
       this.isModified = false;
