--- minecraft/src/net/minecraft/unknown/ChunkCoordinates.java	2018-01-21 00:00:18.000000000 -0800
+++ src/main/java/net/minecraft/unknown/ChunkCoordinates.java	2018-01-21 00:25:14.488602769 -0800
@@ -1,6 +1,6 @@
 package net.minecraft.unknown;
 
-public class ChunkCoordinates implements Comparable {
+public class ChunkCoordinates implements Comparable<ChunkCoordinates> {
    public int x;
    public int y;
    public int z;
@@ -33,7 +33,7 @@
       return this.x + this.z << 8 + this.y << 16;
    }
 
-   public int compareChunkCoordinate(ChunkCoordinates var1) {
+   public int compareTo(ChunkCoordinates var1) {
       if (this.y == var1.y) {
          return this.z == var1.z ? this.x - var1.x : this.z - var1.z;
       } else {
