--- minecraft/src/net/minecraft/unknown/EnumOSMappingHelper.java	2018-01-20 21:02:54.000000000 -0800
+++ src/main/java/net/minecraft/unknown/EnumOSMappingHelper.java	2018-01-20 21:41:14.802041507 -0800
@@ -4,6 +4,9 @@
 
 // $FF: synthetic class
 public class EnumOSMappingHelper {
+
+   public static int[] enumOSMappingArray;
+
    static {
       try {
          enumOSMappingArray[EnumOS2.linux.ordinal()] = 1;
