--- minecraft/src/net/minecraft/unknown/J_ArrayNodeContainer.java	2018-01-21 00:00:32.000000000 -0800
+++ src/main/java/net/minecraft/unknown/J_ArrayNodeContainer.java	2018-01-21 00:01:14.559361633 -0800
@@ -1,6 +1,9 @@
 package net.minecraft.unknown;
 
 class J_ArrayNodeContainer implements J_NodeContainer {
+   public J_JsonListenerToJdomAdapter field_27293_b;
+   public J_JsonArrayNodeBuilder field_27294_a;
+
    J_ArrayNodeContainer(J_JsonListenerToJdomAdapter var1, J_JsonArrayNodeBuilder var2) {
       this.field_27293_b = var1;
       this.field_27294_a = var2;
