--- minecraft/src/net/minecraft/unknown/J_JsonNodeList.java	2018-01-21 00:00:40.000000000 -0800
+++ src/main/java/net/minecraft/unknown/J_JsonNodeList.java	2018-01-21 13:05:57.248172378 -0800
@@ -3,7 +3,9 @@
 import java.util.ArrayList;
 
 final class J_JsonNodeList extends ArrayList {
-   J_JsonNodeList(Iterable var1) {
+   public Iterable<J_JsonNode> field_27405_a;
+
+   J_JsonNodeList(Iterable<J_JsonNode> var1) {
       this.field_27405_a = var1;
 
       for(J_JsonNode var3 : this.field_27405_a) {
