--- minecraft/src/net/minecraft/unknown/NetClientHandler.java	2018-01-21 00:00:32.000000000 -0800
+++ src/main/java/net/minecraft/unknown/NetClientHandler.java	2018-01-21 12:27:44.617855453 -0800
@@ -1,6 +1,7 @@
 package net.minecraft.unknown;
 
 import java.io.BufferedReader;
+import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.InetAddress;
 import java.net.Socket;
@@ -21,7 +22,12 @@
 
    public NetClientHandler(Minecraft var1, String var2, int var3) {
       this.mc = var1;
-      Socket var4 = new Socket(InetAddress.getByName(var2), var3);
+      Socket var4 = null;
+      try {
+          var4 = new Socket(InetAddress.getByName(var2), var3);
+      } catch (IOException e) {
+         e.printStackTrace();
+      }
       this.netManager = new NetworkManager(var4, "Client", this);
    }
 
