--- minecraft/src/net/minecraft/unknown/RailLogic.java	2018-01-21 00:00:38.000000000 -0800
+++ src/main/java/net/minecraft/unknown/RailLogic.java	2018-01-21 00:40:08.702454303 -0800
@@ -4,6 +4,7 @@
 import java.util.List;
 
 class RailLogic {
+   private BlockRail rail;
    private World worldObj;
    private int trackX;
    private int trackY;
@@ -20,7 +21,7 @@
       this.trackZ = var5;
       int var6 = var2.getBlockId(var3, var4, var5);
       int var7 = var2.getBlockMetadata(var3, var4, var5);
-      if (BlockRail.isPoweredBlockRail((BlockRail)Block.blocksList[var6])) {
+      if (((BlockRail)Block.blocksList[var6]).getIsPowered()) {
          this.isPoweredRail = true;
          var7 &= -9;
       } else {
@@ -120,7 +121,7 @@
       return false;
    }
 
-   private int getAdjacentTracks() {
+   public int getAdjacentTracks() {
       int var1 = 0;
       if (this.isMinecartTrack(this.trackX, this.trackY, this.trackZ - 1)) {
          ++var1;
