--- minecraft/src/net/minecraft/unknown/RenderSorter.java	2018-01-21 00:00:32.000000000 -0800
+++ src/main/java/net/minecraft/unknown/RenderSorter.java	2018-01-21 12:19:39.681849385 -0800
@@ -2,14 +2,14 @@
 
 import java.util.Comparator;
 
-public class RenderSorter implements Comparator {
+public class RenderSorter implements Comparator<WorldRenderer> {
    private EntityLiving baseEntity;
 
    public RenderSorter(EntityLiving var1) {
       this.baseEntity = var1;
    }
 
-   public int doCompare(WorldRenderer var1, WorldRenderer var2) {
+   public int compare(WorldRenderer var1, WorldRenderer var2) {
       boolean var3 = var1.isInFrustum;
       boolean var4 = var2.isInFrustum;
       if (var3 && !var4) {
@@ -19,7 +19,9 @@
       } else {
          double var5 = (double)var1.distanceToEntitySquared(this.baseEntity);
          double var7 = (double)var2.distanceToEntitySquared(this.baseEntity);
-         if (var5 < var7) {
+         if (var5 == var7) {
+            return 0;
+         } else if (var5 < var7) {
             return 1;
          } else if (var5 > var7) {
             return -1;
