--- minecraft/src/net/minecraft/unknown/RenderLiving.java	2018-01-21 00:00:28.000000000 -0800
+++ src/main/java/net/minecraft/unknown/RenderLiving.java	2018-01-21 12:31:32.175727048 -0800
@@ -16,6 +16,11 @@
       this.renderPassModel = var1;
    }
 
+   @Override
+   public void doRender(Entity var1, double var2, double var4, double var6, float var8, float var9) {
+      if (var1 instanceof EntityLiving) doRenderLiving((EntityLiving) var1, var2, var4, var6, var8, var9);
+   }
+
    public void doRenderLiving(EntityLiving var1, double var2, double var4, double var6, float var8, float var9) {
       GL11.glPushMatrix();
       GL11.glDisable(2884);
