--- minecraft/src/net/minecraft/unknown/RenderFallingSand.java	2018-01-21 00:00:28.000000000 -0800
+++ src/main/java/net/minecraft/unknown/RenderFallingSand.java	2018-01-21 12:57:10.269512296 -0800
@@ -9,6 +9,12 @@
       this.shadowSize = 0.5F;
    }
 
+   @Override
+   public void doRender(Entity var1, double var2, double var4, double var6, float var8, float var9) {
+      if (var1 instanceof EntityFallingSand)
+         doRenderFallingSand((EntityFallingSand) var1, var2, var4, var6, var8, var9);
+   }
+
    public void doRenderFallingSand(EntityFallingSand var1, double var2, double var4, double var6, float var8, float var9) {
       GL11.glPushMatrix();
       GL11.glTranslatef((float)var2, (float)var4, (float)var6);
