--- minecraft/src/net/minecraft/unknown/StatsSyncher.java	2018-01-21 00:00:20.000000000 -0800
+++ src/main/java/net/minecraft/unknown/StatsSyncher.java	2018-01-21 11:57:20.724124734 -0800
@@ -8,16 +8,16 @@
 import java.util.Map;
 
 public class StatsSyncher {
-   private volatile boolean field_27438_a = false;
-   private volatile Map field_27437_b = null;
-   private volatile Map field_27436_c = null;
+   public volatile boolean busy = false;
+   public volatile Map field_27437_b = null;
+   public volatile Map field_27436_c = null;
    private StatFileWriter field_27435_d;
-   private File field_27434_e;
-   private File field_27433_f;
-   private File field_27432_g;
-   private File field_27431_h;
-   private File field_27430_i;
-   private File field_27429_j;
+   public File field_27434_e;
+   public File field_27433_f;
+   public File field_27432_g;
+   public File field_27431_h;
+   public File field_27430_i;
+   public File field_27429_j;
    private Session field_27428_k;
    private int field_27427_l = 0;
    private int field_27426_m = 0;
@@ -55,7 +55,7 @@
 
    }
 
-   private Map func_27415_a(File var1, File var2, File var3) {
+   public Map func_27415_a(File var1, File var2, File var3) {
       if (var1.exists()) {
          return this.func_27408_a(var1);
       } else if (var3.exists()) {
@@ -96,12 +96,16 @@
    }
 
    private void func_27410_a(Map var1, File var2, File var3, File var4) {
-      PrintWriter var5 = new PrintWriter(new FileWriter(var3, false));
-
       try {
-         var5.print(StatFileWriter.func_27185_a(this.field_27428_k.username, "local", var1));
-      } finally {
-         var5.close();
+         PrintWriter var5 = new PrintWriter(new FileWriter(var3, false));
+
+         try {
+            var5.print(StatFileWriter.func_27185_a(this.field_27428_k.username, "local", var1));
+         } finally {
+            var5.close();
+         }
+      } catch (Exception e) {
+         e.printStackTrace();
       }
 
       if (var4.exists()) {
@@ -116,21 +120,21 @@
    }
 
    public void func_27418_a() {
-      if (this.field_27438_a) {
+      if (this.busy) {
          throw new IllegalStateException("Can't get stats from server while StatsSyncher is busy!");
       } else {
          this.field_27427_l = 100;
-         this.field_27438_a = true;
+         this.busy = true;
          (new ThreadStatSyncherReceive(this)).start();
       }
    }
 
    public void func_27424_a(Map var1) {
-      if (this.field_27438_a) {
+      if (this.busy) {
          throw new IllegalStateException("Can't save stats while StatsSyncher is busy!");
       } else {
          this.field_27427_l = 100;
-         this.field_27438_a = true;
+         this.busy = true;
          (new ThreadStatSyncherSend(this, var1)).start();
       }
    }
@@ -138,7 +142,7 @@
    public void syncStatsFileWithMap(Map var1) {
       int var2 = 30;
 
-      while(this.field_27438_a) {
+      while(this.busy) {
          --var2;
          if (var2 <= 0) {
             break;
@@ -151,20 +155,20 @@
          }
       }
 
-      this.field_27438_a = true;
+      this.busy = true;
 
       try {
          this.func_27410_a(var1, this.field_27434_e, this.field_27432_g, this.field_27430_i);
       } catch (Exception var8) {
          var8.printStackTrace();
       } finally {
-         this.field_27438_a = false;
+         this.busy = false;
       }
 
    }
 
    public boolean func_27420_b() {
-      return this.field_27427_l <= 0 && !this.field_27438_a && this.field_27436_c == null;
+      return this.field_27427_l <= 0 && !this.busy && this.field_27436_c == null;
    }
 
    public void func_27425_c() {
