--- minecraft/src/net/minecraft/unknown/ThreadStatSyncherSend.java	2018-01-21 00:00:38.000000000 -0800
+++ src/main/java/net/minecraft/unknown/ThreadStatSyncherSend.java	2018-01-21 12:35:39.621852532 -0800
@@ -3,19 +3,22 @@
 import java.util.Map;
 
 class ThreadStatSyncherSend extends Thread {
+   private StatsSyncher field_27232_b;
+   private Map field_27233_a;
+
    ThreadStatSyncherSend(StatsSyncher var1, Map var2) {
       this.field_27232_b = var1;
       this.field_27233_a = var2;
    }
 
-   public void run() {
+   public void run() {/* TODO: Fix this
       try {
          StatsSyncher.func_27412_a(this.field_27232_b, this.field_27233_a, StatsSyncher.func_27414_e(this.field_27232_b), StatsSyncher.func_27417_f(this.field_27232_b), StatsSyncher.func_27419_g(this.field_27232_b));
       } catch (Exception var5) {
          var5.printStackTrace();
       } finally {
          StatsSyncher.func_27416_a(this.field_27232_b, false);
-      }
+      }*/
 
    }
 }
