--- minecraft/src/net/minecraft/unknown/TileEntityRendererPiston.java	2018-01-21 00:00:28.000000000 -0800
+++ src/main/java/net/minecraft/unknown/TileEntityRendererPiston.java	2018-01-21 13:07:07.232382910 -0800
@@ -9,6 +9,11 @@
    public TileEntityRendererPiston() {
    }
 
+   @Override
+   public void renderTileEntityAt(TileEntity var1, double var2, double var4, double var6, float var8) {
+      if (var1 instanceof TileEntityPiston) func_31070_a((TileEntityPiston) var1, var2, var4, var6, var8);
+   }
+
    public void func_31070_a(TileEntityPiston var1, double var2, double var4, double var6, float var8) {
       Block var9 = Block.blocksList[var1.getStoredBlockID()];
       if (var9 != null && var1.func_31008_a(var8) < 1.0F) {
